registry/tests/settings.spec.ts

```diff
@@ -263,6 +263,34 @@ describe(url, () => {
             }
         });
 
+        it('should return settings by domain name in case domain name is not root and it does not have own csp config', async () => {
```

Comment:
seems that case I've expected to see another one, something like

`should return default setting value if setting supports domain override but not overridden for passed domain`

Reply:
Thank you. The description is more appropriate. Updated